### PR TITLE
Grade school exercise

### PR DESCRIPTION
**Pull request by macta**
@bencoman @samWson - this is an interesting exercise, its the first one that hints at objects. But I've deliberately not created a Student object as it didn't warrant it for a name, grade pair.

### CODE REVIEW

File: exercises/grade-school/.meta/solution/GradeSchool.class.st [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+]
+
+{ #category : #exercism }
+GradeSchool >> addStudents: aCollection [
+	^ self formatResult: (self addAllStudents: aCollection)
+]
+
+{ #category : #exercism }
+GradeSchool >> desiredGrade: gradeInteger [
+	"Anser the Collection of students filtered by grade, alphabetically"
```
Comment (samWson) [inline exercises/grade-school/.meta/solution/GradeSchool.class.st]: Typo here: `Anser` instead of `Answer`.

**Comment by samWson** [comment]
> this is an interesting exercise, its the first one that hints at objects

Kind of a bit funny when everything is already an object. Looks good to me. I had a look at the Ruby version for comparison. They are using a Hash internally for the students but the actual student could be anything. Looks like your version is storing them as an association which is a good enough object for the lightweight requirements of the exercise.

File: exercises/grade-school/.meta/solution/GradeSchool.class.st (hunk 1 above)
Comment (samWson) [reply exercises/grade-school/.meta/solution/GradeSchool.class.st]: Looks like you have already merged. This isn't critical and can be fixed later.

**Comment by macta** [comment]
Thanks @samWson - I've corrected that. Hopefully we can crank out a few more quite rapidly and get us closer to launch position. I've got a few active users on Discord that are enjoying it, and giving some helpful feedback (so far nothing tragic).

**Pull request merged**